res/linux/mixxx-usb-uaccess.rules

```diff
@@ -56,6 +56,3 @@ KERNEL=="hidraw*", ATTRS{idVendor}=="054c", TAG+="uaccess"
 
 # Missing:
 # - American Musical Supply (AMS/Mixars)
-
-# Only some distribuions require the below
-KERNEL=="hiddev*", NAME="usb/%k", GROUP="uaccess"
```

Comment:
I think the correct rule would have `TAG+="uaccess"` instead, but I'm not a udev expert.

Reply:
Thanks for the indepth explanation